components/automate-ui/src/app/components/admin-sidebar/admin-sidebar.component.html

```diff
@@ -26,8 +26,8 @@
       <chef-sidebar-entry route="/settings/tokens" icon="vpn_key">API Tokens</chef-sidebar-entry>
     </app-authorized>
 
-    <div *ngIf="(iamMajorVersion$ | async) !== 'v1'">
-      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible">Access Management</div>
+    <div *ngIf="(iamMajorVersion$ | async) === 'v2'">
```

Comment:
Better to keep the not-equal-v1 because if we go to v3, etc. we likely want it still to work.

Reply:
LOL -- just refreshed and saw your comment on the very same thing!

Reply:
So I will add in that not-equal-v1 is also consistent with all the other uses in the front-end.